_posts/2019-04-10-aws-dashboard-console-using-arduino.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: Building AWS Dashboard Console using Arduino
+author: m1kma
+excerpt: Arduino compatible microcontrollers are an easy and fun way to create electronic projects. In this blogpost I will cover details of the Arduino project to create a AWS dashboard console prototype.  
+tags:
+- Arduino
+- Electronics
+- AWS
+---
+The system dashboard plays an important role when a real-time system status monitoring and rapid detection of error situations are required. It presents a general view of the running system including metrics, alarms and historical timeseries data. In addition, the dashboard could present special information such as a deployment pipeline status, which is useful especially for the software developers.
```

Comment:
In addition, the dashboard... kuulostaa oudolta mun korvaan. Mun mielestä pitäisi olla In addition, a dashboard...

Ja a deployment pipeline status -> artikkeli pois.

Reply:
timeseries --> time series/time-series ?